osu.Game/Overlays/Social/Header.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Game.Overlays.SearchableList;
+using OpenTK.Graphics;
+using osu.Framework.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics;
+using osu.Framework.Allocation;
+using System.ComponentModel;
+
+namespace osu.Game.Overlays.Social
+{
+    public class Header : SearchableListHeader<SocialTab>

```

Comment:
might be worth nesting this class inside `SocialOverlay`. i tend to prefer nesting generic-named classes like `Header` where possible.